client_cli/src/main.rs

```diff
@@ -544,7 +544,7 @@ mod account {
     impl RunArgs for ListPermissions {
         fn run(self, cfg: &ClientConfiguration) -> Result<Box<dyn Serialize>> {
             let client = Client::new(cfg)?;
-            let find_all_permissions = FindPermissionTokensByAccountId { id: self.id.into() };
+            let find_all_permissions = FindPermissionTokensByAccountId::new(self.id);
```

Comment:
it's a good question how this passes in the CI

Reply:
Very likely a misunderstanding between `cargo` and build scripts. We always assume that you're working from the crate root (that's why CI passed). We also had `cargo-all-features` which entered all subdirs and ran tests from there, but it was done manually and I stopped doing that when we had too much work in October.

Reply:
Created #3311